docs/docker.md

```diff
@@ -2,55 +2,103 @@
 
 # Docker
 
-This page list various operations that can be automated _via_ Docker.
+Production and development images for [Cozy Stack](https://cozy.io), Dockerfiles, docker-compose.yml & co lives [there](https://github.com/cozy/cozy-stack/tree/master/scripts)
```

Comment:
[nitpick] When we have a link to cozy-stack, we try to make it go to https://github.com/cozy/cozy-stack or https://docs.cozy.io/en/cozy-stack/README/. And links to https://cozy.io are just labelled "Cozy".

Reply:
Ok